tests/compat.py

```diff
@@ -13,9 +10,5 @@
 except ImportError:
     cStringIO = None
 
-if is_py3:
-    def u(s):
-        return s
-else:
-    def u(s):
-        return s.decode('unicode-escape')
+def u(s):
```

Comment:
Should we go through and remove all uses of `u(...)` in the tests to simplify them?

Reply:
I left the definition in case it's in use externally, but yes, it should be removed from all of our tests. I'll do a second pass if I missed some.

Reply:
Perhaps add a deprecation warning for any external use?